src/utils/url-helper.js

```diff
@@ -8,6 +8,9 @@
  * See https://swift.org/CONTRIBUTORS.txt for Swift project authors
 */
 
+// we should consider moving `normalizeAssetUrl` here and renaming it since it
```

Comment:
100% agree :) I initially added it because the assets coming from docc had to be prefixed in cases where you have a custom baseUrl, hence the name 🤣

Reply:
I agree. I was going to do it but decided to just add the comment since I don't want to overcomplicate the existing PR with more refactoring.